src/index.html

```diff
@@ -11,7 +11,131 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"    method="post"
+    >
+      <fieldset class="wrapper">
+        <legend>Personal information</legend>
+        <label class="inputs">
```

Comment:
this is like item of fieldset, i think you can name using this informations. `inputs` doesn't seem like good name